test/e2e/command/botkube.go

```diff
@@ -83,6 +83,10 @@ func (c *context) testBotkubeCommand(t *testing.T) {
 				"  - statefulsets\n" +
 				"  - storageclasses\n",
 		},
+		"BotKube notifier showconfig": {
+			command:  "notifier showconfig",
+			expected: "<@U023BECGF> notifier showconfig",
```

Comment:
@anoopmsivadas The expected response is not valid here. `@BotKube notifier showconfig` should return the resource config in the response. https://www.botkube.io/usage/#view-botkube-configuration. I am wondering how the test is passing though

Reply:
@PrasadG193 hey what should I do next

Reply:
@anoopmsivadas I think BotKube is taking some time to respond. Could you try adding some sleep (around 5/10s) before `lastSeenMsg := c.GetLastSeenSlackMessage()`?

Reply:
@PrasadG193 I tried that already, but it didn't help

Reply:
Can you post the output after adding sleep?

Reply:
`assert.Equal(t,"test", m.Attachments)` here I added "test" to see the actual output.
```shell 
            --- FAIL: TestRun/command/Test_BotKube_command/BotKube_notifier_showconfig (11.00s)
                botkube.go:119: 
                        Error Trace:    botkube.go:119
                        Error:          Not equal: 
                                        expected: string("test")
                                        actual  : []slack.Attachment([]slack.Attachment(nil))
                        Test:           TestRun/command/Test_BotKube_command/BotKube_notifier_showconfig
```

and the message string part passes with `expected: "<@U023BECGF> notifier showconfig"`